docusaurus/docs/dev-docs/cli.md

```diff
@@ -356,14 +356,17 @@ Generate [TypeScript](/dev-docs/typescript.md) typings for the project schemas.
 strapi ts:generate-types
 ```
 
-* **strapi ts:generate-types --verbose**<br/>
-  Generate typings with the verbose mode enabled, displaying a detailed table of the generated schemas.
-* **strapi ts:generate-types --silent** or **strapi ts:generate-types -s**<br/>
-  Generate typings with the silent mode enabled, completely removing all the logs in the terminal.
-* **strapi ts:generate-types --out-dir &#60;path&#62;** or **strapi ts:generate-types -o &#60;path&#62;**<br/>
+- **strapi ts:generate-types --debug**<br />
+  [DEPRECATED] **strapi ts:generate-types --verbose**<br/>
```

Comment:
@pwizla I wasn't sure this was the best way to format the deprecation since I didn't have any examples to go by. We're replacing 'verbose' with 'debug' but don't want to remove 'verbose ' until v5 to prevent a breaking change. Feel free to update in any way that looks better.

Reply:
It looks good. Thanks, Ben! It's the first time we use "deprecated" in the docs, so no specific format has been defined yet.